wordpress-mobile/WordPress-iOS
# Issues/7272 bump analytics

Refs #7272 
This PR implements our new analytic keys on the new login view controllers.  The viewed analytics should always be bumped when a view appears as the goal of these analytics is to help us better understand the flow taken through the login screens.  Other analytics are are bumped as needed.

As part of the work on this PR I double checked that the analytics keys used by the previous sign in implementation (e.g. .openedLogin, .signedIn, etc.) are still being bumped as expected.

To test:
- Confirm the "viewed" analytics are being bumped when the new login controllers appear. A breakpoint should be sufficient.
- Confirm that the forgotpassword stat is being bumped when the forgot password buttons on the wpcom password screen, and self-hosted username/password screen are clicked.

Needs review: @nheagy
